pageserver/src/tenant/timeline.rs

```diff
@@ -884,7 +891,12 @@ impl Timeline {
     pub async fn shutdown(self: &Arc<Self>, freeze_and_flush: bool) {
         debug_assert_current_span_has_tenant_and_timeline_id();
 
+        // Signal any subscribers to our cancellation token to drop out
+        tracing::info!("Cancelling CancellationToken");
```

Comment:
```suggestion
        tracing::debug!("Cancelling CancellationToken");
```

Part of normal shutdown, do not want to output this for every tenant.